amp_conf/htdocs/admin/libraries/Console/Chown.class.php

```diff
@@ -149,6 +149,9 @@ protected function execute(InputInterface $input, OutputInterface $output, $quie
 			$this->modfiles['framework'][] = array('type' => 'execdir',
 			'path' => $AMPBIN,
 			'perms' => 0755);
+			$this->modfiles['framework'][] = array('type' => 'dir',
```

Comment:
Upon further thought, I agree this change should be dropped.  Under linux, I guess AMPSBIN defaults to /usr/sbin, which has oodles of system stuff in it, and FreePBX ought not play with the directory permissions.

On FreeBSD, a new directory is created, /usr/local/freepbx/sbin.  Thie directory creation now happens in the install.php script, and an appropriate chmod is done at the time of creation.  So there's really no need to to have Chown.php touch that directory at all.